src/components/RegistrationList.tsx

```diff
@@ -66,6 +77,22 @@ export const RegistrationListItemContent = ({
   const publicationDate = displayDate(entityDescription?.publicationDate);
   const heading = [typeString, publicationDate].filter(Boolean).join(' — ');
 
+  const isPromotedPublication = promotedPublications.includes(id);
+
+  const isMutating = useIsMutating({ mutationKey }) > 0;
+
+  const mutatePromotedPublications = useMutation({
+    mutationKey,
+    mutationFn: (newPromotedPublications: string[]) =>
+      updatePromotedPublications(userCristinId, newPromotedPublications),
+    onSuccess: async (newData) => {
```

Comment:
her gikk vi bort fra async/await
```suggestion
    onSuccess: (newData) => {
```